lib/ansible/modules/cloud/docker/docker_service.py

```diff
@@ -511,18 +511,17 @@ def make_redirection_tempfiles():
 
 
 def cleanup_redirection_tempfiles(out_name, err_name):
-    get_redirected_output(out_name)
-    get_redirected_output(err_name)
+    for i in [out_name, err_name]:
+        os.remove(i)
 
 
 def get_redirected_output(path_name):
     output = []
     with open(path_name, 'r') as fd:
         for line in fd:
             # strip terminal format/color chars
-            new_line = re.sub(r'\x1b\[.+m', '', line.encode('ascii'))
+            new_line = re.sub(r'\x1b\[.+m', '', to_text(line))
             output.append(new_line)
-    fd.close()
```

Comment:
Okay, I just tested this and it looks like fd.close() does not cause an error.  It's useless to have it there but not strictly necessary to remove it.